int16 x_goal
int16 y_goal
int16 z_goal
int16 x
int16 y
int16 z
---
bool result